src/modules/currency/types.ts

```diff
@@ -0,0 +1,8 @@
+export interface Currency {
+  symbol: string;
+  name: string;
+}
+
+export interface GetCurrencyListOutput {
+  [currencyCode: string]: Currency;
```

Comment:
let's avoid keeping generic type to variables.. using specific types will give better developer experience.